pkg/cmd/developer.go

```diff
@@ -6,14 +6,15 @@ import (
 )
 
 type DeveloperCmd struct {
-	Start     dev.StartCmd     `cmd:"" group:"developer" help:"start aserto-one instance"`
-	Stop      dev.StopCmd      `cmd:"" group:"developer" help:"stop aserto-one instance"`
-	Status    dev.StatusCmd    `cmd:"" group:"developer" help:"status of aserto-one instance"`
-	Update    dev.UpdateCmd    `cmd:"" group:"developer" help:"download the latest aserto onebox image"`
-	Console   dev.ConsoleCmd   `cmd:"" group:"developer" help:"launch web console"`
-	Configure dev.ConfigureCmd `cmd:"" group:"developer" help:"configure a policy"`
-	Install   dev.InstallCmd   `cmd:"" group:"developer" help:"install aserto onebox"`
-	Uninstall dev.UninstallCmd `cmd:"" group:"developer" help:"uninstall aserto onebox, removes all locally installed artifacts"`
+	Start               dev.StartCmd            `cmd:"" group:"developer" help:"start aserto-one instance"`
+	Stop                dev.StopCmd             `cmd:"" group:"developer" help:"stop aserto-one instance"`
+	Status              dev.StatusCmd           `cmd:"" group:"developer" help:"status of aserto-one instance"`
+	Update              dev.UpdateCmd           `cmd:"" group:"developer" help:"download the latest aserto onebox image"`
+	Console             dev.ConsoleCmd          `cmd:"" group:"developer" help:"launch web console"`
+	Configure           dev.ConfigureCmd        `cmd:"" group:"developer" help:"configure a policy"`
+	GenerateFromOpenAPI dev.GenerateFromOpenAPI `cmd:"" group:"developer" help:"generate an open api policy"`
```

Comment:
Maybe a nit pick but consider renaming the command to `PolicyFromOpenAPI` or even `PolicyFromSpec`. The current name doesn't really communicate what is being generated.

I think that seeing `aserto developer policy-from-spec http://example.com/openapi.json peoplefinder` is clearer than `aserto developer generate-from-open-api http://example.com/openapi.json peoplefinder`.